python/lang/correctness/exit.yaml

```diff
@@ -4,9 +4,8 @@ rules:
   - python
   message: Use `sys.exit` over the python shell `exit` built-in. `exit` is a helper for the interactive shell and may not
     be available on all Python implementations. https://stackoverflow.com/questions/6501121/difference-between-exit-and-sys-exit-in-python
-  patterns:
-  - pattern: exit(...)
-  - pattern-not: sys.exit(...)
+  pattern: exit($X)
```

Comment:
Test is failing because of this. We still need to explicitly not match `sys.exit(...)` because of the `import sys; exit()` pattern.

Reply:
👍